### DISCUSSION TITLE
Mi opinión sobre el teletrabajo: una crítica a la presentación de la DotNet 2022 de Unai Zorrilla y Cristina González

### DISCUSSION THREAD

**Discussion by montyclt** [iokode/blog#23] [Comments]
Llevo desde antes del inicio de la pandemia trabajando casi al 100% en remoto, y esto es algo para mí que es innegociable en cualquier oferta de empleo, pues invertí mucho tiempo y dinero en montar un despacho para ello, y a día de hoy es el lugar desde donde más cómodo trabajo.

![Una foto de mi despacho de trabajo, con mis tres monitores](https://user-images.githubusercontent.com/9193841/182221500-10fd6e3f-dc39-45aa-968b-274c3cf9ba11.jpeg)

Recientemente, se celebró en Madrid la [DotNet 2022](https://dotnet2022.com/), conferencia técnica a la que tuve el placer de asistir, y [desvirtualizar a mucha gente del sector que conocía por Internet](https://twitter.com/montyclt/status/1541881267268059138).

En la presentación de dicha conferencia, [Unai](https://twitter.com/_unaizc_) nos contó que no sería emitida en directo (aunque sí grabada) para promover la asistencia personal, debido a una serie de “problemas” que explica que ocasionaría el trabajo en remoto.

Una de las primeras cosas que dijo fue que ninguno tenemos una sala especializada con quince pantallas, si no que estás en mitad de la casa, con la familia, no es un sitio especializado en el trabajo. Otra cosa que comentó es que en el trabajo en remoto se trabaja muchas más horas que antes, estás aburrido, no sales a tomar algo…

Todo esto son generalizaciones, mis compañeros de trabajo en VADAVO son mis amigos, probablemente sea uno de mis mejores grupos de amigos. El trabajar en remoto al 100% no ha afectado de forma negativa a esa amistad, y muchos viernes, después del trabajo (desde mi despacho) acudo a cenar con ellos cerca de la oficina. No hago más horas que antes –al contrario, no malgasto una hora en transporte que luego tengo libre para, por ejemplo, escribir este artículo–.

Sencillamente, no puedo estar más en desacuerdo con las afirmaciones que Unai dijo en la presentación. Son generalizaciones que, quizás puede darse en algunos casos, tal vez incluso en la mayoría, pero desde luego no en todos, y el mío es uno de esos casos donde esto no se da.

Me gustaría hacer una petición especial a mi compañero de trabajo @Angeling3, quien trabaja también en remoto 100% desde otro país para que dé su opinión y cuente su experiencia sobre el trabajo 100% remoto.

## El networking
Una de las cosas que Unai dijo sobre los eventos presenciales, es que estos sirven para hacer networking, conocer a gente. Para mí, esto es sin lugar a duda lo que mayor valor da a este tipo de eventos. Las charlas las puedo ver luego en YouTube, pero el networking es más complicado.

Quiero dejar claro que se puede hacer networking online, yo personalmente [he conocido a personas en el chat en directo de una transmisión](https://twitter.com/patrilikestar), con quienes luego he hecho amistad. Yo he conocido a [grandes amigos en los foros de SceneBeta.com](https://twitter.com/duhowpi). Todo eso es networking.

Sin embargo, el networking presencial facilita ciertas cosas frente al networking online. En esta edición, no sólo desvirtualicé a gente muy guay –recuerdo cuando [Sergio Barriel](https://twitter.com/sergiobarriel) se bajó de una moto, se dirigió a mí y me preguntó “¿Eres Ivan Montilla?”–, sino que además conocí y entablé una relación cordial con algunos miembros de la empresa [Voxel](http://voxelgroup.net), lo que me abre puertas a hacer negocios con ellos si en el fututo fuese del interés de ambas partes.

## Las comunidades
Unai utilizó este discurso para promover la participación de distintas comunidades locales y con ello dar paso a [Cristina](https://twitter.com/crisgherrero), quien presentó distintas comunidades técnicas locales. El hecho de que se promueva la participación en comunidades locales me parece algo genial, pero desde luego no creo "demonizar" el trabajo en remoto sea precisamente la forma más adecuada para ello.

**Comment by Angeling3** [comment]
Agradezco la invitación @montyclt.

Uno de mis mayores retos al empezar el trabajo totalmente remoto fue justo la división entre mi tiempo y el del trabajo, dado que ambos ocurren dentro del mismo espacio, con frecuencia llegaban a mezclarse dándome la sensación de estar trabajando más de la cuenta. No fue hasta que tome mayor disciplina en definir estos tiempos aunado con poner un par de reglas en casa y hacer entender a mi familia lo perjudicial que son para mi trabajo las interrupciones, entonces empecé a gozar del mismo tiempo adicional que mencionas y aun mejor, estoy más presente para atender inmediatamente verdaderas urgencias o sencillamente permitirme viajar con mi familia sin desatender mis labores como recordarás hace unos meses.

En cuanto al networking concuerdo totalmente, sin duda considero es la mayor desventaja del trabajo remoto, para nada es lo mismo la interacción presencial que permite conectar con mas personas de manera casual e incluso inesperada, algo que me llegó a ocurrir con cierta frecuencia en mi anterior trabajo, y al día de hoy conservamos muy buena amistad.

Yo lo veo más como una cuestión de preferencia, al final uno decide si mayormente remoto o presencial desde la elección de la vacante, ambos tienen sus claros pros y contras, dependerá de cada quien cuál le resulte mejor en el momento. Independientemente de eso, me parece excelente la intención de que se promuevan los eventos presenciales pero no creo que sea lo correcto justificarlo teniendo que estigmatizar el trabajo remoto.

Por mi parte estoy encantado en tener que trabajar de esta forma porque puedo organizar mi tiempo para no tener una rutina monótona, además gracias a ello es que tengo la oportunidad de colaborar con VADAVO desde México, algo que no creo hubiese sido posible de alguna otra forma al menos por ahora.

Muy chulo el espacio de trabajo, espero algún día también tengamos la oportunidad desvirtualizarnos Ivan. Un saludo y a la espera de la siguiente entrada.

**Reply by montyclt** [reply]
Hola @Angeling3, gracias por tu respuesta.

El tema de las interrupciones, yo lo he vivido en la oficina cuando trabajaba de forma presencial, pero desde que tengo un despacho en el que estoy sólo, es algo del pasado. Gracias a no tener esas interrupciones, me siento mucho más productivo.

El tema del networking creo que no me has entendido bien con lo que he querido decir, no me refería a hacer networking con los compañeros de oficina, que como ya he comentado anteriormente, el trabajo en remoto no me lo ha impedido ni ha empeorado la comunicación ni la amistad que tengo con ellos, pues tenemos herramientas para comunicarnos y registrar lo que hablamos de forma efectiva, y en cuanto a la amistad, nada impide que los viernes vaya después de trabajar a tomar algo con ellos o a cenar.

A mi también me gustaría tener algún día la oportunidad de desvirtualizarnos ;)

Un saludo.

**Comment by montyclt** [comment]
@unaizorrilla me respondió a esta entrada [en este tweet](https://twitter.com/_unaizc_/status/1554233236334809088). Le pedí si podía copiar la respuesta aquí para tenerlo junto a la entrada, pero probablemente no lo vio, así que me he tomado la libertad de copiarla por mi mismo aquí y responderle.

> Gracias por la mención.. sigo creyendo lo mismo no obstante! Hacer cherry picking para argumentar algo es complicado, me alegro que tu caso sea tan positivo, yo tb tengo notas positivas, donde trabajo somos 100% remoto y así seguirá por lo que decir que “demonízanos” el teletrabajo me parece algo sacado de contexto , no?

**Reply by montyclt** [reply]
Hola Unai, gracias por tu respuesta.

Como bien dije en el post “quizás puede darse en algunos casos, tal vez incluso en la mayoría, pero desde luego no en todos”. Mi intención tampoco es negar el hecho, que he conocido en compañeros, de trabajar en el comedor con el portátil, con la TV, las mascotas y la familia, sin llegar a poder hacer nada de forma medianamente productiva. Es simplemente recalcar, que esto no tiene porque ser así.

Tal vez me he pasado un poco empleando la palabra “demonizar”, he modificado la entrada poniéndola entre comillas para “rebajar” un poco su connotación.
